action/template_test.go

```diff
@@ -19,30 +21,45 @@ func TestTemplate(t *testing.T) {
 	defer func() { log.Stdout = o }()
 
 	// TOML
-	Template("", tpl, val)
+	Template("", tpl, val, false)
 	if out.String() != "Hello World!\n" {
 		t.Errorf("Expected Hello World!, got %q", out.String())
 	}
 
+	// force
+	err := Template(tpl, val, "", false)
+	if err == nil {
+		t.Errorf("Expected error but got nil \n")
```

Comment:
don't need a `\n` character here. also, like above it would be helpful to create this using `errors.New(...)` in a global, non-exported `var` so you don't have to allocate a new one. finally, can you give a more descriptive error message in here?